plugins/lsp_java.lua

```diff
@@ -0,0 +1,43 @@
+-- mod-version:3 -- lite-xl 2.1
+
+local common = require "core.common"
+local config = require "core.config"
+local lsp = require "plugins.lsp"
+
+local installed_path_plugin = USERDIR .. PATHSEP .. "plugins" .. PATHSEP .. "lsp_java"
+local installed_path_library = USERDIR .. PATHSEP .. "libraries" .. PATHSEP .. "jdk"
+
+local platform
+if PLATFORM == "Windows" then
+  platform = "win"
+elseif PLATFORM == "Mac OS X" then
+  platform = "mac"
+else
+  platform = "linux"
+end
+
+local java_home = installed_path_library .. PATHSEP .. "jdk-21.0.2"
+local java_bin = java_home .. PATHSEP .. "bin" .. PATHSEP .. "java"
```

Comment:
Should add `.exe` on Windows, right?

Reply:
Let's see.

Reply:
Yes. I'll add it.

Reply:
This `local java_bin = java_home .. PATHSEP .. "bin" .. PATHSEP .. "java" .. (PLATFORM == "Windows" and ".exe" or "")` should be enough I think.